engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -1031,16 +1031,16 @@ struct devouring_plague_t final : public priest_spell_t
       double old_multiplier   = cast_state( old_s )->rolling_multiplier;
 
       // figure out how many old ticks to roll over
-      int num_full_ticks = as<int>( std::floor( ( old_remains - time_to_tick ) / old_tick ) );
+      double num_ticks = std::floor( ( old_remains - time_to_tick ) / old_tick );
```

Comment:
Probably call this "num_munched_ticks" or something, to make it clear that those are just the ticks being added as damage.